### DISCUSSION TITLE
Nextjsv13, deploying to vercel error, but on localhost works fine

### DISCUSSION THREAD

**Discussion by EthanHook2** [vercel/community#1643] [Help]
### Summary
<html>
<body>

Error occurred prerendering page "/". Read more: https://nextjs.org/docs/messages/prerender-error
--
21:43:05.590 | TypeError: fetch failed
21:43:05.590 | at Object.fetch (node:internal/deps/undici/undici:11118:11)
21:43:05.591 | at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
21:43:05.591 | at async fetchPageInfo (/vercel/path0/.next/server/pages/index.js:1068:17)
21:43:05.591 | at async getStaticProps (/vercel/path0/.next/server/pages/index.js:994:22)
21:43:05.591 | at async renderToHTML (/vercel/path0/node_modules/next/dist/server/render.js:386:20)
21:43:05.592 | at async /vercel/path0/node_modules/next/dist/export/worker.js:304:36
21:43:05.592 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
21:43:05.592 | info  - Generating static pages (3/3)
21:43:05.594 |  
21:43:05.594 | > Build error occurred
21:43:05.597 | Error: Export encountered errors on following paths:
21:43:05.597 | /
21:43:05.597 | at /vercel/path0/node_modules/next/dist/export/index.js:404:19
21:43:05.597 | at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
21:43:05.597 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
21:43:05.598 | at async /vercel/path0/node_modules/next/dist/build/index.js:1229:21
21:43:05.598 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
21:43:05.598 | at async /vercel/path0/node_modules/next/dist/build/index.js:1091:17
21:43:05.598 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
21:43:05.598 | at async Object.build [as default] (/vercel/path0/node_modules/next/dist/build/index.js:65:29)
21:43:05.663 | Error: Command "npm run build" exited with 1

</body>
</html>

Keep getting this error everytime I go to deploy to vercel,. it works fine on my localhost but i just get erros when i go to deploy it. Anyhelp would much appricated, Id be happy to share the repo if needed.

**Comment by amyegan** [comment]
Hi @EthanHook2! Can you share your repo or a minimal reproduction so we can get a better look at the problem? The document at https://nextjs.org/docs/messages/prerender-error has some general suggestions to try, but I need to know more to be able to suggest a specific solution

**Reply by EthanHook2** [reply]
Hi, not sure if this will alow you to open the repo; 

https://github.com/EthanHook2/Portfolio-demo1/tree/main

If not I will make it public when I get home. 

Thanks
